### PR TITLE
fix(TDI-38807):revert the feature and just throw an exception.

### PR DESCRIPTION
**Pull request by pyzhou-talend**
**What is the current behavior?** (You can also link to an open issue here)

https://jira.talendforge.org/browse/TDI-38807

**What is the new behavior?**

**Please check if the PR fulfills these requirements**

- [ ] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/492/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/492/)

File: components/components-azurestorage/src/test/java/org/talend/components/azurestorage/utils/AzureStorageUtilsTest.java [hunk 1]
```diff
@@ -23,9 +23,15 @@
 
 import org.junit.Before;
 import org.junit.Test;
+import org.talend.daikon.i18n.GlobalI18N;
+import org.talend.daikon.i18n.I18nMessages;
 
 public class AzureStorageUtilsTest {
 
+    private static final String Except = null;
```
Comment (wwang-talend) [inline components/components-azurestorage/src/test/java/org/talend/components/azurestorage/utils/AzureStorageUtilsTest.java]: where to use it?

Comment (pyzhou-talend) [inline components/components-azurestorage/src/test/java/org/talend/components/azurestorage/utils/AzureStorageUtilsTest.java]: It's generated by mistake,I will remove it.

File: components/components-azurestorage/src/main/resources/org/talend/components/azurestorage/utils/messages.properties [hunk 2]
```diff
@@ -1,3 +1,4 @@
 error.FileNotExist=No match file "{0}" exist!
 #
-error.InvalidSAS=Shared Acces Signature is invalid.
\ No newline at end of file
+error.InvalidSAS=Shared Acces Signature is invalid.
+error.invalidDirectory=Local folder should be a valid directory.
```
Comment (wwang-talend) [inline components/components-azurestorage/src/main/resources/org/talend/components/azurestorage/utils/messages.properties]: folder is a directory, seems that it have nonsense, not sure.

Comment (pyzhou-talend) [inline components/components-azurestorage/src/main/resources/org/talend/components/azurestorage/utils/messages.properties]: I can change to "A valid directory should be filled in Local folder option"

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/526/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/526/)

**Pull request merged**